src/Yoti/ActivityDetails.php

```diff
@@ -18,12 +18,14 @@ class ActivityDetails
     const ATTR_GIVEN_NAMES = 'given_names';
     const ATTR_FULL_NAME = 'full_name';
     const ATTR_DATE_OF_BIRTH = 'date_of_birth';
+    const ATTR_AGE_VERIFIED = 'age_verified';
```

Comment:
How was this working before?

Reply:
Basically that attribute is not coming from connect, the age derivation value is extracted by a regex. This constant is mainly for the plugins. I won't have to define it for each plugin.